product/reports/670_Performance by Asset Type - Middleware Servers/170_Web Metrics - every minute over last hour.yaml

```diff
@@ -0,0 +1,61 @@
+---
+where_clause:
+dims:
+created_on: 2016-10-09 14:13:30.822035 Z
+reserved:
+title: "Web Metrics - every minute over the last hour"
+conditions:
+updated_on: 2016-10-09 17:23:33.732482 Z
```

Comment:
2016? please see my other comment about the dates

Reply:
They are not necessary. So I remove them in both reports